File: nom-api/Nom.Orch/Models/Household/HouseholdResponseModel.cs
// File: Nom.Orch/Models/Household/HouseholdResponseModel.cs

namespace Nom.Orch.Models.Household
{
    public class HouseholdResponseModel
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public long HouseholdGroupId { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? ModifiedDate { get; set; }

        // Members
        public List<HouseholdMemberResponseModel>? Members { get; set; }

        // Statistics
        public int MemberCount { get; set; }
        public int RecipeCount { get; set; }
        public int PlanCount { get; set; }
        public int ShoppingListCount { get; set; }
    }
} 

File: nom-ui/src/app/core/models/household-response.model.ts
import { HouseholdMemberResponseModel } from './household-member-response.model';

export interface HouseholdResponseModel {
  id: number;
  name: string;
  description: string | null;
  householdGroupId: number;
  createdDate: string;
  modifiedDate: string | null;
  members: HouseholdMemberResponseModel[] | null;
  memberCount: number;
  recipeCount: number;
  planCount: number;
  shoppingListCount: number;
}
